test/dev-should-serve.unit.js

```diff
@@ -0,0 +1,46 @@
+import path from 'path';
+import test from 'ava';
+import * as buildUtils from '@now/build-utils';
+import shouldServe from '../src/commands/dev/lib/default-should-serve';
+
+test('shouldServe on should-serve-nodejs', async (t) => {
+  const cwd = path.resolve(__dirname, 'fixture/unit/should-serve-nodejs');
+  const files = await buildUtils.glob('**', cwd);
+
+  t.is(shouldServe({
+    files,
+    entrypoint: 'index.js',
+    requestPath: 'index.js'
+  }), true);
+
+  t.is(shouldServe({
+    files,
+    entrypoint: 'index.js',
+    requestPath: '/'
+  }), true);
+
+  // This is same behavior as in production
```

Comment:
This behavior is considered a bug, but we should wait and replicate how we would fix it in production. For now, just keep the behavior (and implementation too I guess) like production

Reply:
Agreed, we need to replicate production 👍